<launch>

  <node pkg="loam_velodyne" type="LoopOptimizing" name="LoopOptimizing" output="screen" >
  </node>

</launch>
